backend/api/v1/endpoints/auth_email.py

```
@@ -0,0 +1,78 @@
+    message['subject'] = "Your OTP for Verification"
+    
+    # Send the email using Gmail API
+    try:
+        send_message = service.users().messages().send(userId="me", body={'raw': message.as_string()}).execute()
+        print(f"Message sent successfully: {send_message['id']}")
+    except Exception as e:
+        print(f"Error sending email: {e}")
+
+# Main route for OTP email request (example)
+@app.post("/send-otp/{email}")
+async def send_otp(email: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `send_otp`

(ANN201)